src/js/menu.js

```diff
@@ -101,7 +95,7 @@ var showSensorsMenu = function(thermostat) {
     }
   });
 
-  menu = new UI.Menu({
+  var sensorMenu = new UI.Menu({
```

Comment:
It will still return to main screen if you perform an action. This change is just around using the back button.